src/engraving/libmscore/beam.cpp

```diff
@@ -1345,7 +1345,7 @@ void Beam::layout2(const std::vector<ChordRest*>& chordRests, SpannerSegmentType
             int bottomLastLine = bottomLast ? bottomLast->upNote()->line() : 0;
             if (chordRests.size() == 2 && chordRests[0]->staffMove() != chordRests[1]->staffMove()) {
                 // if there are only two notes, one on each staff, special case
-                qreal horizDistance = chordRests[1]->stemPosX() - chordRests[0]->stemPosX();
+                //qreal horizDistance = chordRests[1]->stemPosX() - chordRests[0]->stemPosX();
```

Comment:
Commenting it out only, as I'm unsure how/whether it relates to the comment above

Reply:
@asattely ?

Reply:
Or does @Nick-Mazuk know something about this, as this change is a contination of #10130?

Reply:
I don't see that code anywhere in that file anymore, so I can't be of much help.

@Jojo-Schmitz for future reference, you can use `git blame` to see who added the code.